redis/utils.py

```diff
@@ -36,3 +36,41 @@ def str_if_bytes(value):
 
 def safe_str(value):
     return str(str_if_bytes(value))
+
+
+def dict_merge(*dicts):
+    """
+    Merge all provided dicts into 1 dict.
+    """
+    merged = {}
+
+    for d in dicts:
```

Comment:
I'm torn. I really don't think you should have to sanitise inputs to a class that merges dicts, and should be able to remove the _if not isinstance_ check. Is there a strong reason you feel the need?

Reply:
removed